tests/test_utils.py

```diff
@@ -14,6 +19,7 @@ class TestIO(unittest.TestCase):
     def setUp(self) -> None:
         """Set up."""
         self.msdf = parse_sssom_table(f"{data_dir}/basic.tsv")
+        self.msdf2 = parse_sssom_table(f"{data_dir}/basic7.tsv")
```

Comment:
can you please make better names for these test data and variables?

Reply:
@cthoyt if this is important lets make a separate issue about this, we have 25 tests written this way now, and I don't think we need to start now.

Reply:
(keep PRs small)